### ISSUE TITLE
Console closes when executing throw without a catch

### ISSUE THREAD

**Issue by Zamlox** [red/red#1842]
Executing:
```
red>> throw 10
```
will close the console.

**Comment by x8x** [comment]
Looks like fixed. I get this on MacOS and Linux:
```
  throw 10
*** Throw error: no catch for throw: 10
*** Where: =
```

**Comment by dockimbel** [comment]
Fixed by commit [7bf5ccb7](https://github.com/red/red/commit/a3ba27a192e77fc0629c0b9848fcd2c37bf5ccb7).

Will investigate the wrong `Where:` field reporting.

**Comment by dockimbel** [comment]
The error report is more accurate now:
```
red>> throw 10
*** Throw error: no catch for throw: 10
*** Where: throw
```
